### ISSUE TITLE
How to include all files in wiki's subfolders also?

### ISSUE THREAD

**Issue by iegik** [adriantanasa/github-wiki-sidebar#13]
```
Home.md
FAQ/QA1.md
FAQ/QA2.md
About.md
```

running `npx github-wiki-sidebar --silent` will see only following:

```
Home
About
```

**Comment by neotrow** [comment]
@iegik 

Did you ever find a solution for this?

**Comment by iegik** [comment]
@neotrow Yes, take a look https://github.com/iegik/CivOne/wiki/FAQ~How-to-simply-fork-the-Wiki-page#wikis-structure

> To avoid issues on some OS, we use `~` as folder separator and `-` to replace spaces in file names.
